pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -693,7 +696,7 @@ func (p *PriorityQueue) PendingPods() []*v1.Pod {
 	for _, pInfo := range p.unschedulablePods.podInfoMap {
 		result = append(result, pInfo.Pod)
 	}
-	return result
+	return result, fmt.Sprintf(pendingPodsSummary, p.activeQ.Len(), p.podBackoffQ.Len(), len(p.unschedulablePods.podInfoMap))
```

Comment:
Return string value directly seems a little dull, but considering it's used for debugging, I think it's ok.
/lgtm

Reply:
I thought of returning a `map[string][]*v1.Pod` for PendingPods(), but seems a bit too much given it's for debugging purposes.